apps/okreads-e2e/src/specs/search-books.spec.ts

```diff
@@ -1,27 +1,17 @@
 import { $, $$, browser, ExpectedConditions } from 'protractor';
 
 describe('When: Use the search feature', () => {
-  it('Then: I should be able to search books by title', async () => {
```

Comment:
Why did you delete the old test?

Reply:
Hi Katerina,

Thanks for the review. Appreciate your time!

To answer your question, since we have removed search button functionality as this had no relevance after implementing this task - therefore test case was removed from here.